src/main/java/edu/greenblitz/tobyDetermined/commands/NodeToNeighbourSupplier.java

```diff
@@ -0,0 +1,36 @@
+package edu.greenblitz.tobyDetermined.commands;
+
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeArm;
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeBase;
+import edu.greenblitz.tobyDetermined.Nodesssss.AStar;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+import java.util.LinkedList;
+import java.util.function.Supplier;
+
+public class NodeToNeighbourSupplier implements Supplier<Command> {
+    private NodeArm start;
+    private NodeArm end;
+    private NodeToNeighbourCommand[] nodeCommands;
+    public NodeToNeighbourSupplier(NodeArm endNode) {
+        end = endNode;
+        SmartDashboard.putNumber("passIsFinished",0);
+    }
+    public Command get() {
+        start = NodeBase.getInstance().getCurrentNode();
+        SmartDashboard.putNumber("starting node",start.getId());
+        SmartDashboard.putNumber("end node",end.getId());
+        LinkedList<NodeArm> path = AStar.getPath(start,end);
+        assert path != null;//can delete
+        if (path == null)//can delete
+            return null;//can delete
```

Comment:
I've heard a rumor that these lines can be deleted

Reply:
deleted